### DISCUSSION TITLE
How can I apply an alignment to the column?

### DISCUSSION THREAD

**Discussion by FelipeBattistotti** [Comcast/react-data-grid#2549] [Q&A]
How can I apply an alignment to the column?

Example: align the contents of a single column to the right.

**Comment by daalgi** [comment]
Try this when you define the columns:
```
const columns = [
    { key: "value", name: "Value",
        formatter: ({ row }) => 
        <div style={{textAlign: "right"}} >{row.value}</div>
    }
]
```

**Comment by FelipeBattistotti** [comment]
> Try this when you define the columns:
> 
> ```
> const columns = [
>     { key: "value", name: "Value",
>         formatter: ({ row }) => 
>         <div style={{textAlign: "right"}} >{row.value}</div>
>     }
> ]
> ```


Thanks @daalgi !
I had already found a solution.
Thanks for the feedback.

Maybe you can help me with this issue?
[https://github.com/adazzle/react-data-grid/issues/2532](https://github.com/adazzle/react-data-grid/issues/2532)

if you can help, I appreciate it.

**Comment by amanmahajan7** [comment]
You can use `column.cellClass` and `column.headerCellClass`
https://github.com/adazzle/react-data-grid/blob/main/src/types.ts#L16
